scripts/pi-hole/php/header.php

```diff
@@ -185,152 +74,43 @@ function getTemperature()
         html { background-color: #000; }
     </style>
 <?php } ?>
-    <link rel="stylesheet" href="style/vendor/SourceSansPro/SourceSansPro.css?v=<?php echo $cacheVer; ?>">
+
+    <!-- Common styles -->
     <link rel="stylesheet" href="style/vendor/bootstrap/css/bootstrap.min.css?v=<?php echo $cacheVer; ?>">
+<?php if ($auth) { ?>
```

Comment:
Do you want to leave this here or move it to `header_authenticated.php`? It won't be used on the login page.

Reply:
This file (`header.php`) is included in both cases.

This `if` is needed to only use this block when authenticated.

Reply:
Yeah this is exactly my point. `header.php` is used for `loginpage.php` and `header_authenthicated.php`. But this part is never used on the `loginpage.php` because `$auth` is not set when the `loginpage` is shown. Therefore it makes sense to me to move it to `header_authenticated` and remove the `if` clause.

Reply:
No.

`header_authenticated.php` is used ONLY for authenticated pages.
`header.php` is used in both cases (it is [included in `header_authenticated.php`](https://github.com/pi-hole/AdminLTE/blob/ac15ecaf90e2f4ca8b9228dd78f259b543c9ec15/scripts/pi-hole/php/header_authenticated.php#L138)).

Reply:
I'm trying to put all `<head>` stuff in one place to make it easy to maintain.
I'm also trying to group all `<scripts>` and all `<link>`  tags together in just one place.

We can remove the `if`s and separate these tags in 2 files, but I think its more logical to put them together.